lib/trino-filesystem-gcs/src/main/java/io/trino/filesystem/gcs/GcsFileSystemConfig.java

```diff
@@ -148,14 +163,92 @@ public GcsFileSystemConfig setJsonKeyFilePath(String jsonKeyFilePath)
         return this;
     }
 
+    public Optional<RetryStrategy> getRetryStrategy()
+    {
+        return this.retryStrategy;
+    }
+
+    @Config("gcs.retry.strategy")
+    @ConfigDescription("Determines the retry strategy: DEFAULT|UNIFORM. See https://cloud.google.com/storage/docs/retry-strategy.")
+    public GcsFileSystemConfig setRetryStrategy(RetryStrategy retryStrategy)
+    {
+        this.retryStrategy = Optional.ofNullable(retryStrategy);
+        return this;
+    }
+
+    public Optional<@MinDuration("0ms") Duration> getInitialRetryDelay()
+    {
+        return initialRetryDelay;
+    }
+
+    @Config("gcs.retry.initial-delay")
+    @ConfigDescription("Initial delay between rpc retries. See https://cloud.google.com/storage/docs/retry-strategy.")
```

Comment:
Let's remove these links. The page doesn't directly explain what the setting does. (though what it does seems fairly obvious and similar to other retry configuration)

Also, capitalize RPC